homeassistant/components/goodwe/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""The Goodwe inverter component."""
+from datetime import timedelta
+import logging
+
+from goodwe import InverterError, RequestFailedException, connect
+
+from homeassistant import config_entries, core
+from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    CONF_MODEL_FAMILY,
+    CONF_NETWORK_RETRIES,
+    CONF_NETWORK_TIMEOUT,
+    DEFAULT_NETWORK_RETRIES,
+    DEFAULT_NETWORK_TIMEOUT,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    KEY_COORDINATOR,
+    KEY_INVERTER,
+    PLATFORMS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
+):
```

Comment:
Can you ensure that all methods have type hints for both the parameters and the return types?
`async_setup_entry` and `async_unload_entry` are missing the return type.

Reply:
sure, will do.